.codegen/service.py.tmpl

```diff
@@ -247,7 +247,17 @@ class {{.Name}}API:{{if .Description}}
         # deduplicate items that may have been added during iteration
         seen = set()
         {{- end}}{{if and .Pagination.Offset (not (eq .Path "/api/2.0/clusters/events")) }}
-        query['{{.Pagination.Offset.Name}}'] = {{if eq .Pagination.Increment 1}}1{{else}}0{{end}}{{end}}
+        query['{{.Pagination.Offset.Name}}'] =
+        {{- if eq .Pagination.Increment 1 -}}
+          1
+        {{- else if contains .Path "/scim/v2/" -}}
```

Comment:
This fails codegen: `function "contains" not defined`. Did you test generating the SDK with this change? Is there a corresponding change in the `deco` tool that is needed as well?